### DISCUSSION TITLE
Skip_TPM_Check_on_Dynamic_Update.cmd No Longer Provides In-place Upgrading with Recent Win 11 ISO builds

### DISCUSSION THREAD

**Discussion by PlateauRealm** [AveYo/MediaCreationTool.bat#296] [General]
I notice that with recent Win11 22631 builds, such as 22631.2787, running the mounted ISO setup.exe with Skip_TPM_Check_on_Dynamic_Update.cmd activated will no longer provide me with the option to keep all existing apps and settings.
Fortunately, I find that in-place upgrading with the Quick_11_iso_esd_wim_TPM_toggle.bat and providing a valid retail key WILL provide me with the option to keep my current apps and settings.

**Comment by AveYo** [comment]
Probably a licensing conflict issue - script is inserting a generic EI.CFG and that might have changed or the handling of retail keys have changed. I will try to replicate it and get back to you.
